src/routes/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { AuthenticaedRoute, UnauthenticaedRoute } from './protectedRoute';
```

Comment:
```suggestion
import { AuthenticatedRoute, UnauthenticatedRoute } from './protectedRoute';
```
there are some typos here.

Reply:
Fixed! 👍